ui/components/app/assets/asset-list/asset-list.tsx

```diff
@@ -37,7 +37,7 @@ const TokenListContainer = React.memo(
           },
         });
       },
-      [],
+      [onClickAsset, primaryCurrencyProperties.suffix, trackEvent],
```

Comment:
Noticed this on assets list and from new ESLint ruleset. Best to actually update this callback on these dependency changes.